include/tvSettings.h

```diff
@@ -1288,6 +1288,48 @@ tvError_t SetDynamicGamma(double tvGammaValue);
  */
 tvError_t GetTVSupportedDolbyVisionModes(tvDolbyMode_t *dvModes[],unsigned short *count);
 
+/**
+ * @brief Gets the supported HDR10 modes and their count
+ *
+ * This function returns the supported HDR10 modes and their count
+ *
+ * @param[out] HDR10Modes[]            - List of available HDR10 modes. Refer ::tvDolbyMode_t @n
+ *                                     Valid values are member of ::tvDolbyMode_t
+ * @param[out] count                - Count of supported HDR10 modes. Max is tvMode_Max and min is 0
+ *
+ * @return tvError_t
+ *
+ * @retval tvERROR_NONE                      - Success
+ * @retval tvERROR_INVALID_PARAM             - Parameter is invalid
+ * @retval tvERROR_INVALID_STATE             - Interface is not initialized
+ * @retval tvERROR_OPERATION_NOT_SUPPORTED   - Operation is not supported
+ * @retval tvERROR_GENERAL                   - Underlying failures - SoC, memory, etc
+ *
+ * @pre TvInit() should be called before calling this API
+ */
+tvError_t GetTVSupportedHDR10Modes(tvDolbyMode_t *HDR10Modes[],unsigned short *count)
```

Comment:
1)General errors are thrown if any issue with failure from pqserver
2) T.C. will be defined.
3)OUT_OF_MEMORY not present in our tvError_t
4) T.C expects it to return INVALID_STATE since it is called before TvInit()

Reply:
removed from this issue since that will be brought in ODM api removals ticket.